tests/test_model.py

```diff
@@ -0,0 +1,110 @@
+from ocetrac.model import Tracker 
+
+import pytest
+import unittest
+import xarray as xr
+import numpy as np
+import scipy.ndimage
+from skimage.measure import regionprops 
+from skimage.measure import label as label_np
+import dask.array as dsa
+
+class TrackTester(unittest.TestCase):
+    
+    def example_anomaly_data(self):
```

Comment:
I think you can relatively easily get rid of unittest here by just defining `example_anomaly_data` as [fixture](https://docs.pytest.org/en/6.2.x/fixture.html) and then adding this as input for all other functions below. E.g. `def test_morphological_operations(example_anomaly_data):`

Reply:
Good suggestion! I'll make this change, which should simplify things.